index.d.ts

```diff
@@ -6,6 +6,9 @@ export class MaxBufferError extends Error {
 	constructor();
 }
 
+type StreamItem = string | Buffer | ArrayBuffer | ArrayBufferView;
+type AnyStream = Readable | ReadableStream<StreamItem> | AsyncIterable<StreamItem>;
```

Comment:
I would export this, in case the user needs to refer to the input without passing it directly.

Reply:
Fixed in 6990861. :+1: